src/trix/models/composition.coffee

```diff
@@ -217,11 +232,14 @@ class Trix.Composition extends Trix.BasicObject
       @removeCurrentAttribute(attributeName)
 
   canSetCurrentAttribute: (attributeName) ->
```

Comment:
This method should condition for text and block attributes like `setCurrentAttribute` does. Doing so will probably clarify the somewhat confusing `switch` statement.